services/movie.py

```diff
@@ -0,0 +1,40 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids and actors_ids:
+        return queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    if genres_ids:
+        return queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        return queryset.filter(actors__id__in=actors_ids)
+    return queryset

```

Comment:
Leave only one `return` and modify `queryset` defined on line 10. Remove the first if condition as well. Remember that querysets are lazy in Django